.coveragerc

```diff
@@ -1,6 +1,6 @@
 [run]
 source = pinax
-omit = pinax/stripe/conf.py,pinax/stripe/tests/*,pinax/stripe/migrations/*
+omit = pinax/stripe/conf.py,pinax/stripe/migrations/*
```

Comment:
I’d like to exclude tests from coverage.

Reply:
Isn't it good to see if they are actually 100% covered, i.e. executed?